examples/servo/src/adapter.rs

```diff
@@ -5,23 +5,28 @@ use std::cell::{Ref, RefCell, RefMut};
 use std::rc::{Rc, Weak};
 
 use servo::{Servo, WebView};
-use slint::ComponentHandle;
 use smol::channel::{Receiver, Sender};
 
+use slint::ComponentHandle;
+
 #[cfg(not(target_os = "android"))]
 use slint::wgpu_27::wgpu;
 
 use crate::{MyApp, WebviewLogic, rendering_context::ServoRenderingAdapter};
 
+/// Upgrades a weak reference to SlintServoAdapter to a strong reference.
+/// Panics if the adapter has been dropped.
 pub fn upgrade_adapter(weak_ref: &Weak<SlintServoAdapter>) -> Rc<SlintServoAdapter> {
```

Comment:
I suggest to inline this function, i.e. replace every use of it with a direct call to `upgrade()`.

(doesn't have to be part of this PR)